_studio/shared/src/cm_mem_copy.cpp

```diff
@@ -3176,7 +3176,7 @@ bool CmCopyWrapper::CheckSurfaceContinuouslyAllocated(const mfxFrameSurface1 &su
     case MFX_FOURCC_NV16:
         {
             mfxU32 stride_in_bytes = surf.Data.PitchLow + ((mfxU32)surf.Data.PitchHigh << 16);
-            size_t luma_size_in_bytes = stride_in_bytes * mfx::align2_value(surf.Info.Height);
+            size_t luma_size_in_bytes = stride_in_bytes * mfx::align2_value(surf.Info.Height, 32);
```

Comment:
These two lines of code are completely identical in all cases. Please pull them out of ```switch```

Reply:
@asmirno1 , will it work if application makes 16 alignment?

Reply:
Done